flaskfront/opgflaskfront/templates/index.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">Flask app</h1>
+      <p class="govuk-body">This is the index page of service front v2 flask app</p>
```

Comment:
More generic as this is the library, so not specific to our "service front"

Reply:
This file now updated and moved to feedback repo